spec/stack.js

```diff
@@ -11,12 +11,48 @@ describe('Stack', () => {
     expect(Stack).to.be.a('function')
   })
 
+  context('length()', () => {
+    it('returns the size of the stack', () => {
```

Comment:
We should test for possible edge cases as well - does `length` work for empty stacks?